### DISCUSSION TITLE
Achar Parceiro para desenvolver IOS e Android

### DISCUSSION THREAD

**Discussion by arthurerl** [frontendbr/forum#1941] [Perguntas e Respostas]
Pessoal estou iniciando no mundo dos apps, e ja estou com algumas ideias para desenvolver, porem somente estou começando então por agora estou focando em desenvolver android e querendo ou não nos dias de hoje, precisa ter uma flexibilidade onde todos apps precisam estar em IOS e Android.

Portando estou a procura de alguém que saiba programar para IOS ou ainda esta aprendendo para que possamos assim fechar uma parceria de desenvolvimento.

Caso eu estiver no lugar errado ou alguém souber onde posso procurar isso me avisem por favor.

Fico no aguardo,

**Comment by wilsonneto-swe** [comment]
Não consigo te ajudar muito rs Mas uma ótima pedida para acelerar teu desenvolvimento para apps "mid-tier" é Flutter ou React Native, depende de vários fatores, mas sempre que preciso desenvolver um app sigo para um destes, e se eu fiz o site em react minha tendência é ir de react native e reaproveitar a maior parte do código (lógica, consumo de apis, entidades e etc).

Sucesso ai!

**Reply by giovannism20** [reply]
Pensei em sugerir a mesma coisa.

**Comment by ninetails** [comment]
Agora que vi, não tem comunidade de desenvolvedores mobile no GitHub? Que o ideal seria ver em algum lugar desses, não?

Ou talvez comunidades em Telegram ou Discord?

Sei que devem ter desenvolvedores mobile aqui, mas por não ser muito o foco não sei se conseguirá encontrar muitos ativos por aqui.
